xknx/io/connectionstate.py

```diff
@@ -7,18 +7,22 @@
 class ConnectionState(RequestResponse):
     """Class to send ConnectonStateRequest and wait for ConnectionStateResponse."""
 
-    def __init__(self, xknx, udp_client, communication_channel_id):
+    def __init__(self, xknx, udp_client, communication_channel_id, net_bind):
```

Comment:
```suggestion
    def __init__(self, xknx, udp_client, communication_channel_id, net_bind=None):
```
make it a keyword attribute like in the other 2 classes